seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/AbstractSeleniumJSRunner.scala

```diff
@@ -28,6 +28,13 @@ abstract class AbstractSeleniumJSRunner(browserProvider: SeleniumBrowser,
     _console = console
   }
 
+  protected def ignoreKeepAlive: Boolean = {
+    val name = code.name
+    name == "frameworkDetector.js" ||
+    name == "testFrameworkInfo.js" ||
+    name == "testMaster.js"
```

Comment:
It would be possible if we change add a `ignoreKeepAlive` parameter to `JSRunner.run` and `AsyncJSRunner.start` in `scala-js/js-envs`. But this would mean braking binary compatibility with older versions. I could start by doing the changes on `scala-js/js-envs` and only merge this PR on the next release. WDYT?